src/global/graphqls/SignOut.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from 'apollo-boost';
+
+export interface SignOutResponse {
```

Comment:
I think this is wrong response , right? Please check the response in the "Network" tab when You clicked on the logout. It's somethink like logout: boolean